apps/common-app/src/apps/reanimated/examples/ShareablesExample.tsx

```diff
@@ -169,11 +169,13 @@ function BooleanDemo() {
   const expectedStatus: Status = 'ok';
 
   const handlePress = () => {
-    const bool = true;
+    const boolTrue = true;
+    const boolFalse = false;
     runOnUI(() => {
       'worklet';
       try {
-        if (bool === true) {
+        const checks = [boolTrue === true, boolFalse === false];
+        if (checks.every(Boolean)) {
```

Comment:
No need to be this generic
```suggestion
        if (boolTrue === true && boolFalse === false) {
```